### PR TITLE
Created Bundle skeleton

### PR DESCRIPTION
**Pull request by alongosz**
This PR provides initial skeleton of the Bundle, without any implementations of specific features.

Please see https://jira.ez.no/browse/EZP-28868 for the  purpose of this package. I plan to make another PR implementing it.

Here I just need another pair of eyes ;)

### CODE REVIEW

**Comment by alongosz** [comment]
@bdunogier @Nattfarinn @webhdx just a skeleton, any objections here?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# eZ Platform Standard Design Bundle
+
+This Bundle defines standard Design and Theme to be handled by [ezplatform-design-engine](https://github.com/ezsystems/ezplatform-design-engine).
+
+Standard Design is the default Design used by the templates shipped with eZ Platform.
+
+## Installation
+
+1. In your eZ Platform 2.1+ project, require this package by the Composer.
```
Comment (bdunogier) [inline README.md]: "with Composer"

File: src/bundle/EzPlatformStandardDesignBundle.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformStandardDesignBundle;
+
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+/**
+ * eZ Platform Standard Design Bundle.
+ *
+ * This bundle defines standard Design and Theme for ezplatform-design-engine.
```
Comment (bdunogier) [inline src/bundle/EzPlatformStandardDesignBundle.php]: I don't think I'd repeat that here.

**Comment by alongosz** [comment]
Thanks! Fixed.

**Pull request merged**